CMakeLists.txt

```diff
@@ -45,11 +45,11 @@ find_package(Wayland REQUIRED client server egl)
 find_package(WaylandScanner REQUIRED)
 find_package(WPE 1.5.90 REQUIRED)
 
-add_compile_definitions(
-    WPE_FDO_COMPILATION
-    G_LOG_DOMAIN=\"WPE-FDO\"
-    _FILE_OFFSET_BITS=64
-    _LARGEFILE64_SOURCE=1
+add_definitions(
+    -DWPE_FDO_COMPILATION
+    -DG_LOG_DOMAIN=\"WPE-FDO\"
+    -D_FILE_OFFSET_BITS=64
+    -D_LARGEFILE64_SOURCE=1
```

Comment:
I am not a fan of `add_definitions()` but we don't have a better option, so be it.

(Maybe the bots could be switched to Meson, hehe :wink:)